requirements.txt

```diff
@@ -2,3 +2,4 @@
 ### Database                                        ###
 #######################################################
 dj-database-url
+django-phonenumber-field
```

Comment:
ik zou alle requirements pinnen op een versie, dan loopt straks je live omgeving niet achter op je dev omgeving of andersom.